zebROS_ws/src/field_obj_tracker/src/convert_coords.cpp

```diff
@@ -3,9 +3,12 @@
 #include "field_obj_tracker/convert_coords.h"
 #include <ros/ros.h>
 
-ConvertCoords::ConvertCoords(const image_geometry::PinholeCameraModel &model)
-  : model_(model)
+private:
+  image_geometry::PinholeCameraModel model_;
+
```

Comment:
This shouldn't be needed here since it was added in the class definition in the .h file.